geode-redis/src/main/java/org/apache/geode/redis/internal/RegionProvider.java

```diff
@@ -210,7 +210,7 @@ public boolean removeKey(ByteArrayWrapper key, RedisDataType type, boolean cance
           return stringsRegion.remove(key) != null;
         } else if (type == RedisDataType.REDIS_HLL) {
           return hLLRegion.remove(key) != null;
-        } else if (type == RedisDataType.REDIS_LIST) {
+        } else if (type == RedisDataType.REDIS_LIST || type == RedisDataType.REDIS_SORTEDSET) {
```

Comment:
Do we have stories in the backlog about sorted sets? Looks like we missed this earlier.

Reply:
Added a placeholder for sorted sets.